app/controllers/ips_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
   def create
     @ip = current_organisation.ips.new(ip_params)
     if @ip.save
-      flash[:notice] = 'IP Added: ' + @ip.address
+      flash[:notice] = "#{@ip.address} added, it will be active starting tomorrow"
       redirect_to ips_path(anchor: 'ips')
```

Comment:
You can one line this if it will fit, but it may be too long a line. 
```
  redirect_to ips_path(anchor: 'ips'), notice: "#{@ip.address} added, it will be active starting tomorrow"
```

Reply:
Same as [above](https://github.com/alphagov/govwifi-admin/pull/106#discussion_r214708023).

Reply:
Fixed in https://github.com/alphagov/govwifi-admin/pull/106/commits/aea325075babddb8d5918ceecc43df08484f8839